File: HW01/PYTHON/project/CampingBag.py
from .OutdoorItem import OutdoorItem

class CampingBag(OutdoorItem):
    def __init__(self, price, size, thickness):
        super().__init__(price)
        self.size = size
        self.thickness = thickness
        
    def display(self):
        print(f"${self.price} - This camping bag is size {self.size} and has a thickness of {self.thickness} inches", end="")

File: HW01/CPP/src/CampingBag.cpp
#include "../include/CampingBag.h"
#include <iostream>
#include <string>
using namespace std;

CampingBag::CampingBag(string size, double thickness, double price) : OutdoorItem(price){
    this->size = size;
    this->thickness = thickness;
    this->price = price;
}

void CampingBag::display() {
    cout << "$" << this->price << " - This camping bag is size " << this->size << " and has a thickness of " << this->thickness << " inches";
}